api/swagger.yaml

```diff
@@ -52,17 +52,17 @@ paths:
     get:
       tags:
         - Event
-      summary: Get the latest event matching the parameters
+      summary: Get the latest event matching the required query parameters
       parameters:
         - name: keptnContext
           in: query
           type: string
-          required: false
+          required: true
```

Comment:
Are you sure that **always** both, the `keptnContext` as well as the `type` are required fields?
Would it be a valid use case to only have the latest event of e.g. a specific `keptnContext`?

Reply:
This is intentional.